internal/pkg/services/load-balancer/utils/utils.go

```diff
@@ -0,0 +1,104 @@
+package utils
+
+import (
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+)
+
+type LoadBalancerClient interface {
+}
+
+func GetDefaultPayload() *loadbalancer.CreateLoadBalancerPayload {
```

Comment:
make this a const (or multiple ones) of the `generatepayload` pkg instead of these getters in a separate `utils` pkg. See the `argus scrape-config generate-payload` implementation for a similar case

Reply:
moved the getter results in a var in generate payload, utils have been removed